code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/ContentCard.java

```diff
@@ -22,7 +22,10 @@
  * A {@link ContentCard} object encapsulates the information necessary for a non-disruptive yet
  * interactive offer. Customers can use the Messaging SDK to render the content card in a
  * pre-defined format or implement their own rendering.
+ *
+ * @deprecated Use {@link ContentCardSchemaData} instead
```

Comment:
Can we make the same change in `FeedItem.java` as well?

Reply:
done